app/backend/src/couchers/servicers/api.py

```diff
@@ -216,9 +217,7 @@ def Ping(self, request, context, session):
         )
 
     def GetUser(self, request, context, session):
-        user = session.execute(
-            select(User).where_users_visible(context).where_username_or_id(request.user)
-        ).scalar_one_or_none()
+        user = session.execute(select(User).where_username_or_id(request.user)).scalar_one_or_none()
```

Comment:
You will also need to modify the functions below to get lite user and plurals (getliteusers, etc)